nemo/collections/asr/data/audio_to_text_dataset.py

```diff
@@ -546,8 +630,35 @@ def get_audio_to_text_char_dataset_from_config(
         )
         return dataset
 
+    # Instantiate a code-switched dataset if config is present
+    if config.get('is_code_switch', False):
```

Comment:
maybe "is_code_switched" as the name here and some other params is a little more user friendly?

Reply:
Versus code_switch_*?

Reply:
I have changed it to `is_code_switched` as requested